tests/js/server/dispatcher-and-inline-tests.js

```diff
@@ -7,15 +7,29 @@ var gpii = fluid.registerNamespace("gpii");
 
 var path = require("path");
 
-var jqUnit  = fluid.require("jqUnit");
+var jqUnit  = fluid.require("node-jqunit");
 var request = require("request");
 
 require("gpii-express");
-require("../../");
+require("../../../index");
+require("../lib/promiseWrapper");
 
-var viewDir = path.resolve(__dirname, "../views");
+var viewDirs = [
```

Comment:
This is now done.  For convenience, the express grade resolves all raw views entries into a member variable.